src/components/Checkout/SearchBar.tsx

```diff
@@ -21,6 +21,9 @@ const SearchBar: React.FC<SearchBarProps> = ({ data, setSearchData, setSearchAct
     }
     setSearchTerm(e.target.value);
     filterFunction(e.target.value);
+    // reset scroll position
```

Comment:
Normally I am not a big fan of most comments in code (I prefer [self describing ](https://en.wikipedia.org/wiki/Self-documenting_code) code.
This is different, as it adds to the code. Thank you!